library/ST7735/__init__.py

```diff
@@ -98,19 +98,23 @@
 ST7735_YELLOW = 0xFFE0  # 0b 11111 111111 00000
 ST7735_WHITE = 0xFFFF  # 0b 11111 111111 11111
 
-
-def color565(r, g, b):
```

Comment:
Did you intend to delete this?

Reply:
Yes, sorry.

I thought that it was legacy code from a prior version of `image_to_data` since there was no public method that receives a RGB565 color and the conversion is being handled with bit manipulation in numpy.

Should I restore it?

(Or delete the references in the tests... I didn't see the tests :facepalm: .)